docs/release/3.4/release-3-4-24-2.md

```diff
@@ -0,0 +1,92 @@
+OSG Software Stack -- Data Release -- 3.4.24-2
+==============================================
+
+**Release Date**: 2019-03-05
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   CA Certificates based on [IGTF 1.96](http://dist.eugridpma.info/distribution/igtf/current/CHANGES)
+    -   withdrawn superseded QuoVadis-Grid-ICA (1st gen) CA (BM)
+    -   added new trust anchor MD-Grid-CA-T for rollover of existing CA (MD)
+    -   discontinued expiring 2009 series MD-Grid-CA (MD)
```

Comment:
Can we capitalize these sub-bullets?

Reply:
I just blindly copy these from the IGTF ChangeLog. This time the didn't capitalize. Fixed.